target_snowflake/__init__.py

```diff
@@ -129,8 +130,28 @@ def persist_lines(config, lines, table_cache=None, file_format_type: FileFormatT
             raise Exception(f"Line is missing required key 'type': {line}")
 
         t = o['type']
-
-        if t == 'RECORD':
+        if t == 'FASTSYNC':
+            
+            stream_row_count = 0 
+            if 'files' in o:
+                stream = o['stream'] 
+
+                for filename in o['files']:
+                    file = os.path.join('fastsync', filename)
+                    size_bytes = os.path.getsize(file)
+                     
+                    with gzip.open(file, 'rb') as f:
+                        for i, l in enumerate(f):
+                            pass
+
+                    count = i + 1
+                    stream_row_count = stream_row_count + count
```

Comment:
Is this being used for anything?

Reply:
same as above. It's been a while since i looked at this, so if it needs to be rewritten i can do so

Reply:
nevermind, the stream_row_count was unused.  Trashed it